ledger-tool/src/main.rs

```diff
@@ -327,9 +339,18 @@ fn output_ledger(
 
     if method == LedgerOutputMethod::Json {
         stdout().write_all(b"\n]}\n").expect("close array");
+    } else {
+        output_sorted_program_ids("Summary of Programs", all_program_ids);
     }
 }
 
+fn output_sorted_program_ids(format: &str, program_ids: HashMap<Pubkey, u64>) {
+    let mut program_ids_array: Vec<_> = program_ids.into_iter().collect();
+    // Sort descending by count of program id
+    program_ids_array.sort_by(|a, b| b.1.cmp(&a.1));
+    println!("{}: {:?}", format, program_ids_array);
```

Comment:
```suggestion
    println!("{}:", format);
    for (program_id, count) in program_ids_array.iter() {
        println!("{:<44}: {}", program_id.to_string(), count);
    }
```

columns for readability